BUILDING.md

```diff
@@ -198,44 +196,57 @@ RUN apt-get update && \
     apt-get install -y libseccomp-dev
 ```
 
-In our Docker container we will build `runc` build, which includes
-[seccomp](https://en.wikipedia.org/wiki/seccomp), [SELinux](https://en.wikipedia.org/wiki/Security-Enhanced_Linux),
-and [AppArmor](https://en.wikipedia.org/wiki/AppArmor) support. Seccomp support
-in runc requires `libseccomp-dev` as a dependency (AppArmor and SELinux support
-do not require external libraries at build time). Refer to [RUNC.md](docs/RUNC.md)
-in the docs directory to for details about building runc, and to learn about
-supported versions of `runc` as used by containerd.
+Let's suppose you've built an image named `containerd/build` from the above `Dockerfile`.
 
-Let's suppose you build an image called `containerd/build` from the above Dockerfile. You can run the following command:
+You can run the following command:
 
 ```sh
-docker run -it --privileged \
-    -v /var/lib/containerd \
-    -v ${PWD}/runc:/go/src/github.com/opencontainers/runc \
-    -v ${PWD}/containerd:/go/src/github.com/containerd/containerd \
-    -e GOPATH=/go \
-    -w /go/src/github.com/containerd/containerd containerd/build sh
+docker run -it \
+    -v ${PWD}/containerd:/src/containerd \
+    -v ${PWD}/runc:/src/runc \
+    -w /src/containerd \
+    containerd/build
 ```
 
 This mounts both `runc` and `containerd` repositories in our Docker container.
 
-From within our Docker container let's build `containerd`:
+From within the Docker container, let's build `containerd`:
 
 ```sh
-cd /go/src/github.com/containerd/containerd
 make && make install
 ```
 
-These binaries can be found in the `./bin` directory in your host.
-`make install` will move the binaries in your `$PATH`.
+You can check the installed binaries with:
+
+```sh
+$ which containerd
+/usr/local/bin/containerd
+
+$ containerd --version
+containerd github.com/containerd/containerd/v2 v2.0.0-rc.3-189-ga406da962.m a406da962818ba9ca8f38b4ac5215753ceb1846c.m
+```
 
 Next, let's build `runc`:
 
 ```sh
-cd /go/src/github.com/opencontainers/runc
+cd /src/runc
 make && make install
 ```
 
+You can check the installed binaries with:
+
+```sh
+$ which runc
+/usr/local/sbin/runc
+
+$ runc --version
+runc version 1.2.0-rc.2+dev
+commit: v1.2.0-rc.2-38-g2ee5564b
```

Comment:
This should use a stable version

Reply:
Sorry, but I don't understand your comment.

I've just followed [the existing instructions in `BUILD.md`](https://github.com/containerd/containerd/blob/f5d5407c2ff12865653a9a132d5783196be82763/BUILDING.md?plain=1#L189) that said to `git clone` `runc` and compile it.

Do you mean the docs should say to e.g. clone the latest release tag of `runc` instead, or something like that?

Thanks!

Reply:
Yes

Reply:
Done, I've updated the doc.